vkcs/networking/resource_anycastip/spec.json

```diff
@@ -0,0 +1,155 @@
+{
+    "provider": {
+        "name": "vkcs"
+    },
+    "resources": [
+        {
+            "name": "anycastip",
+            "description": "Manages a Anycast IP resource within VKCS.",
+            "schema": {
+                "attributes": [
+                    {
+                        "name": "id",
+                        "string": {
+                            "computed_optional_required": "computed",
+                            "description": "ID of the anycast ip."
+                        }
+                    },
+                    {
+                        "name": "region",
+                        "string": {
+                            "computed_optional_required": "computed_optional",
+                            "description": "The region in which to obtain the Networking client. If omitted, the `region` argument of the provider is used. Changing this creates a new resource.",
+                            "plan_modifiers": [
+                                {
+                                    "custom": {
+                                        "imports": [
+                                            {
+                                                "path": "github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+                                            }
+                                        ],
+                                        "schema_definition": "stringplanmodifier.RequiresReplaceIfConfigured()"
+                                    }
+                                }
+                            ]
+                        }
+                    },
+                    {
+                        "name": "name",
+                        "string": {
+                            "computed_optional_required": "computed_optional",
+                            "description": "Name of the anycast ip."
+                        }
+                    },
+                    {
+                        "name": "description",
+                        "string": {
+                            "computed_optional_required": "computed_optional",
+                            "description": "Description of the anycast ip."
+                        }
+                    },
+                    {
+                        "name": "network_id",
+                        "string": {
+                            "computed_optional_required": "required",
+                            "description": "ID of the external network to choose ip for anycast ip from."
+                        }
+                    },
+                    {
+                        "name": "subnet_id",
+                        "string": {
+                            "computed_optional_required": "computed",
```

Comment:
does not backend allow to set it? in contrasting to fip api?

Reply:
Yes, only admins can set subnet_id (and we expose user-only api in terraform)